placeSpecificContent/hq/courtyard/controlPanel/controlPanel_server.js

```diff
@@ -0,0 +1,406 @@
+//
+// controlPanel_server.js
+// 
+// Created by Rebecca Stankus on 07/16/2019
+// Copyright High Fidelity 2019
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+
+(function() {
+    var HALF = 0.5;
+
+    var DATA_RETRIEVAL_TIMEOUT_MS = 500;
+    var PRELOAD_DATA_RETRIEVAL_MAX_ATTEMPTS = 10;
+
+    var DONE_SOUND = SoundCache.getSound(Script.resolvePath("resources/sounds/DONE_SOUND.mp3?1"));
+    var TRANSITION_SOUND = SoundCache.getSound(Script.resolvePath("resources/sounds/TRANSITION_SOUND.mp3?1"));
+    var DONE_VOLUME = 0.2;
+    var TRANSITION_VOLUME = 0.05;
+
+    var INDEX_WHERE_STATE_NAME_BEGINS = 21;
+
+    var NUMBER_OF_BUTTONS = 3;
+    var BUTTON_TO_ENTITY_POSITIONING_MAP = {};
+
+    var FIXED_DECIMAL_PLACES = 2;
+    /* MAP WILL LOOK LIKE THIS BUT BUTTONS MAY NOT BE LISTED IN THIS ORDER: 
+
+    {
+        "{00000000-0000-0000-0000-000000000000}"(id of stage button): {
+            "courtyardPosition": { x: 0, y: 0, z: 0 } (position of courtyard when in the stage state)
+            "roundtablePosition": { x: 0, y: 0, z: 0 } (position of courtyard when in the stage state)
```

Comment:
just a little confused on if what we are asking for is the position of courtyard on all of these based on the name of those object keys

Reply:
Yes, we need the postion of both the courtyard and the roundtable at each these states: stage, courtyard, roundtable